com.github.rphys.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=srb2 rphys
+Comment=Gameplay overhaul mod for Sonic Robo Blast 2
+TryExec=rphys
+Exec=rphys
+Icon=com.github.rphys
+Terminal=false
+Categories=Game;
+Name[en_US]=com.github.rphys.desktop
```

Comment:
This is wrong.

Reply:
Assuming you meant the .desktop, I fixed that

Reply:
No, why is the name, the app-id? It should be the specific name of the application like "srb2 rphys"

Reply:
Ohm I didn't intentionally add that second Name line, it just appeared in the file at some point. I'll delete it,